static/css/src/_base.scss

```diff
@@ -1,28 +1,31 @@
 :root {
-    --heading-font-family: 'Spline Sans Mono', monospace;
-    --body-font-family: 'Spline Sans Mono', monospace;
-    --heading-txt: #000;
-    --body-txt: #4d4d4d;
+    --heading-font-family: "Lexend", sans-serif;
+    --body-font-family: "Nunito", sans-serif;
+    --heading-txt: #5E7581;
+    --body-txt: #5C5C5C;
+    --body-bg: #F8FBFC;
 
-    --ip-border: #d9d9d9;
+    --primary-color: #098CEA;
+    --primary-gradient: linear-gradient(180deg, #60B9F8 0%, #098CEA 100%);
+
+    --ip-border: #A7B7C0;
     --ip-border--focus: #898989;
     --ip-border--disabled: hsla(0, 0%, 85%, 0.5);
-    --ip-bg: #fafafa;
+    --ip-bg: #fff;
```

Comment:
Per the design, the input element's default background is transparent (same as body-bg). Is that not the case?